File: backend/app/schemas/stock.py
from pydantic import BaseModel

class StockSearchItem(BaseModel):
    ts_code: str; symbol: str; name: str; exchange: str; industry: str | None = None; sector: str | None = None

class FundamentalTaskCreate(BaseModel):
    ts_code: str
    report_period: str | None = None
    user_question: str | None = None

File: frontend/src/types/stock.ts
export interface StockBasic{ts_code:string;symbol:string;name:string;exchange:string;industry?:string;sector?:string}
export interface StockSummary{price:number;pct_chg:number;total_mv:number;pe_ttm:number;pb:number;dividend_yield:number;latest_report_period:string}
export interface TimeSeriesPoint{date:string;[key:string]:string|number|null|undefined}

